packages/app/components/layout/sidebar/index.tsx

```diff
@@ -0,0 +1,34 @@
+
+import { SideBar as SideBarUI, Button, ButtonIcon, XStack, YStack, Nav } from "@my/ui";
+import { Link } from "@my/ui";
+import { IconSendLogo, IconSendLogoSmall, IconTelegramLogo, IconTwitterLogo } from "../../icons";
+import { SideBarNavLink } from "./SideBarNavLink";
+import { SideBarFooterLink } from "./SideBarFooterLink";
+import { twitter as twitterSocial, telegram as telegramSocial } from '../../../data/socialLinks'
```

Comment:
```suggestion
import { twitter as twitterSocial, telegram as telegramSocial } from 'app/data/socialLinks'
```

Use the TS paths, young warrior.